python/lsst/ts/hvac/xml/hvac_mqtt_to_SAL_XML.py

```diff
@@ -225,6 +247,33 @@ def get_telemetry_mqtt_topics_and_items():
     return telemetry_mqtt_topics
 
 
+def get_command_mqtt_topics_and_items():
+    """Convenience method to collect all MQTT topics and their items that
+    accept commands.
+
+    Returns
+    -------
+    command_mqtt_topics: `dict`
+        A dictionary with the MQTT topics as keys and their items as value.
```

Comment:
Are MQTT topics strings or instances of HvacTopic?

I strongly suggest taking a pass over your code to look for this ambiguity. Where practical please use the term HvacTopic (or some other enum if there are additional categories of topics) instead of strings. Where you really want topic names (strings), please use the term "topic name".

Reply:
They are strings. I agree that a rework is welcome and I'll do that some time in the not too distant future.